tests/Traits/CreatesTestInvoices.php

```
@@ -8,26 +8,66 @@
+        $invoiceAttrs = [
+            'buyer' => ['name' => 'Test Buyer'],
+            ...$attrs,
+            'items' => [[
+                'title' => 'Test Item',
+                'quantity' => 1,
+                'unit_price' => $actualTotal ?? 10.00,
+                'tax_percentage' => null,
+            ]],
+        ];
+
+        $invoice = Invoice::make($invoiceAttrs);
```

PHPMD (2.15.0)

Avoid using static access to class '\SimpleParkBv\Invoices\Models\Invoice' in method 'createInvoiceWithTotal'. (undefined)

(StaticAccess)